books/centaur/aignet/observability.lisp

```diff
@@ -475,10 +475,20 @@
          :short "List of literals"))
 
 (local
- (defthm nth-of-lit-eval-list
-   (bit-equiv (nth n (lit-eval-list lits invals regvals aignet))
-              (lit-eval (nth n lits) invals regvals aignet))
-   :hints(("Goal" :in-theory (enable nth lit-eval-list)))))
+ (encapsulate
+   ()
+
+   (local
+    (defthm lemma
```

Comment:
Do you know why this lemma is needed? I would have thought the rewrite-quoted-constant rule for lit-fix under lit-equiv would have caused the NIL to normalize to 0, and I think there's probably also a rewrite rule for lit-eval of 0.

Reply:
Hi @solswords - as I think back to what I did here 9 days ago, I recall that without this lemma there was a subgoal with the hypothesis (equal (lit-eval nil invals regvals aignet) 1). The existence of that subgoal suggests that there may not have been a rewrite rule for lit-eval of 0.